cmake/modules/AddSwift.cmake

```diff
@@ -1384,6 +1389,15 @@ function(add_swift_library name)
           endif()
         endforeach()
 
+        set(swiftlib_framework_depends_flattened ${SWIFTLIB_FRAMEWORK_DEPENDS})
```

Comment:
There definitely is a move here: the ICU section and the swiftlib_framework_depends_flattened switched. I do see that it makes a little more sense.